Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -267,13 +267,13 @@ def main():
         args: dict = demisto.args()
         params: dict = demisto.params()
         # There're several options for tenant_id & auth_and_token_url due to the recent credentials set supoort enhancment.
-        tenant_id: str = params.get('tenant_id', '') \
+        tenant_id: str = params.get('creds_tenant_id', {}).get('password', '') \
             or params.get('_tenant_id', '') \
```

Comment:
this moves _tenant_id to have higher priority over tenant_id. was the opposite before. is this intentional?

Reply:
It actually doesn't really matter...  as both `_tenant_id` and `tenant_id` are hidden they cannot be edited by the users.